exasol_data_science_utils_python/schema/dbobject.py

```diff
@@ -25,3 +25,6 @@ def __eq__(self, other):
 
     def __repr__(self):
         return generate_repr_for_object(self)
+
+    def __hash__(self):
+        return hash(self._name)
```

Comment:
same as above, maybe add a safeguard check here to not forget to adjust if needed:
```suggestion
        assert(len(self.__dict__.keys()) == 1) #Update here if necessary
        return hash(self._name)
```

Reply:
replaced with generate_hash_for_object